jena-text/.gitignore

```diff
@@ -0,0 +1 @@
+/text-query.mdtext
```

Comment:
Not needed. Add .gitignore to .gitignore.

Could you attach text-query.mdtext to the JIRA please?